workloads/spec-reference.md

```diff
@@ -0,0 +1,1095 @@
+that represents the intention of delivering to the cluster the configuration
+that are produced by the supply chain.
+
+Parameters:
+
+<table>
+  <tr>
+    <th>parameter name</th>
+    <th>meaning</th>
+    <th>example</th>
+  </tr>
+
+  <tr>
+    <td><code>serviceAccount</code></td>
+    <td>
+      name of the serviceaccount (in the same namespace as the Deliverable) to
+      use for providing the necessary permissions to create the children
+      objects for deploying the objects created by the supply chain to the
+      cluster.
+    </td>
+    <td>
+      <pre>
+      - name: serviceAccount
+        value: default
+      </pre>
+    </td>
+  </tr>
+</table>
+
+> **Note** `--service-account` flag sets the `spec.serviceAccountName` key in
+> the Workload object. To configure the `serviceAccount` parameter, use
+> `--param serviceAccount=...`.
+
+> **Note** On build clusters where a corresponding `ClusterDelivery` doesn't
+> exist, the Deliverable takes no effect (similarly to a Workload without a
+> SupplyChain, no action is taken).
+
+## Deliverable Parameters reference
+
+This section describes the parameters that are provided to the
+Deliverable object, such as what can be set on `deliverable.spec.params`.
+
+The Deliverable is relevant in the context of deploying to a Kubernetes cluster
+that the configuration that is produced through the resources defined by
+a ClusterSupplyChain:
+
+```
+      Workload              (according to ClusterSupplyChain in `build` cluster)
+
+        - fetch source
+        - build
+        - test
+        - scan
+        - generate kubernetes config
+        - push k8s config to git repository / image registry
+
+
+      Deliverable           (according to ClusterDelivery in `run` cluster)
+
+        - fetch kubernetes config (from git repository / image registry)
+        - apply kubernetes objects to cluster
+
+```
+
+In the following section, you find the reference documentation that relates specifically to the
+two resources defined in the `basic` ClusterDelivery part of the
+`ootb-delivery-basic` package:
+
+```
+source-provider                     fetches kubernetes configuration
+    |
+    |  kubernetes configuration
+    |
+app-deploy                          deploys to the cluster the objects in the
+                                    kubernetes configuration fetched
+```
+
+For information about the ClusterDelivery shipped with `ootb-delivery-basic`,
+and the templates used by it, see:
+
+- [Out of the Box Delivery Basic](ootb-delivery-basic.hbs.md)
+- [Out of the Templates](ootb-templates.hbs.md)
+
+For information about the use of the Deliverable object in a multicluster
+environment, see [Getting started with multicluster Tanzu Application
+Platform](../multicluster/getting-started.hbs.md).
+
+For reference information about Deliverable, see [Deliverable and Delivery
+custom resources](https://cartographer.sh/docs/v0.5.0/reference/deliverable/).
+
+## source provider
```

Comment:
Should this be listed in the 'List of supply chain resources' table above?

Reply:
source-provider looks like it's at the top of that table for me.

Reply:
OK **yes** there **is** a 
H3 **source-provider** under H2 **Workload Parameter Reference** and the table correctly links to this

but there is also a H3 **source-provider** under H2 Deliverable Parameter reference

just checking this is all correct - pleas confirm.